bundles/binding/org.openhab.binding.bticino/src/main/java/org/openhab/binding/bticino/internal/BticinoDevice.java

```diff
@@ -264,30 +465,293 @@ else if (p_protocol_read.getProperty("messageType").equalsIgnoreCase("CEN Basic
+                                    eventPublisher.postUpdate(l_item.getName(), StringType.valueOf(state));
+                                }
+                                logger.debug("T_Relay : {} / {}", l_item.getName(), state);
+                            }
+                            break;
+                        case 101:
+                            // Status Control Mode
+                            if (p_protocol_read.getProperty("hStatus").equalsIgnoreCase("2")) {
+                                switch (Integer.parseInt(p_protocol_read.getProperty("what"))) {
+                                    case 1:
+                                        state = "HeatOn";
+                                        eventPublisher.postUpdate(l_item.getName(), StringType.valueOf(state));
+                                        break;
+                                    case 2:
+                                        state = "CoolOn";
+                                        eventPublisher.postUpdate(l_item.getName(), StringType.valueOf(state));
+                                        break;
+                                }
+                                logger.debug("T_ControlMode : {}", l_item.getName());
+                            }
+                            break;
+                        case 102:
+                            // Status Main Unit Remote
+                            if (p_protocol_read.getProperty("hStatus").equalsIgnoreCase("3")) {
+                                logger.debug("T_ControlRemote : {}", l_item.getName());
+                                eventPublisher.postUpdate(l_item.getName(),
+                                        StringType.valueOf(p_protocol_read.getProperty("messageDescription")));
+                            }
+                            break;
+                        case 103:
+                            // Status Main Unit Probe Status
+                            if (p_protocol_read.getProperty("hStatus").equalsIgnoreCase("4")) {
+                                if (hMainCtrlProbeState.length() > 0) {
+                                    hMainCtrlProbeState = hMainCtrlProbeState + ", "
+                                            + p_protocol_read.getProperty("messageDescription");
+                                } else {
+                                    hMainCtrlProbeState = p_protocol_read.getProperty("messageDescription");
+                                }
+                                logger.debug("T_ControlMessage 1: {} / {}", l_item.getName(), hMainCtrlProbeState);
+                                eventPublisher.postUpdate(l_item.getName(), StringType.valueOf(hMainCtrlProbeState));
+                            }
+                            break;
+                        case 104:
+                            // Status Main Unit Failure
+                            if (p_protocol_read.getProperty("hStatus").equalsIgnoreCase("5")) {
```

Comment:
No point in using `equalsIgnoreCase()` here

Reply:
OK
see commit https://github.com/openhab/openhab1-addons/pull/5486/commits/c1c8f3a86ba8641c7333467de21a0ba489322073